git-commitgraph/tests/access/octopus_merges.rs

```diff
@@ -0,0 +1,140 @@
+use crate::{fixture_path, hex_to_id};
+use git_commitgraph::{Graph, GraphPosition};
+
+const ROOT: &[u8] = b"09e3185550fdb352031df65f0e81e74df3b9a556";
+const ROOT_INDEX: GraphPosition = GraphPosition(0);
+const COMMIT1: &[u8] = b"33316871b4ab988ea87ba60aaf0138ddb4fb0d21";
+const COMMIT1_INDEX: GraphPosition = GraphPosition(2);
+const COMMIT2: &[u8] = b"cbfe147d1826a0d2701f7c80dbe0e2cfba385b15";
+const COMMIT2_INDEX: GraphPosition = GraphPosition(6);
+const COMMIT3: &[u8] = b"9dab5d610702ceca4bd889a5ef2662cacfd53178";
+const COMMIT3_INDEX: GraphPosition = GraphPosition(5);
+const COMMIT4: &[u8] = b"2ec748404455c0b0451b186851a815da13918ebb";
+const COMMIT4_INDEX: GraphPosition = GraphPosition(1);
+const THREE_PARENTS: &[u8] = b"7070cd0b59cd46a0f0fa0c50ca9148390ae1e7c7";
+const THREE_PARENTS_INDEX: GraphPosition = GraphPosition(4);
+const FOUR_PARENTS: &[u8] = b"5ca46a72d91817add3a7218f3eaf8b2b6c1caa13";
+const FOUR_PARENTS_INDEX: GraphPosition = GraphPosition(3);
+
+#[test]
+fn v1() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
Sure, I can remove the V1 stuff. Shall I keep the HashKind stuff around? It doesn't provide any value right now either.

Reply:
Thanks! I think HashKind might provide value since [the canonical implementation](https://github.com/git/git/blob/master/commit-graph.c#L47:L47) also seems to have abstracted over the used hash algorithm.
It seems this is something we should do as well, if not done already. Feel free to add what's needed to `HashKind` to support that.